packages/slate-editor/src/extensions/inline-contacts/InlineContactsExtension.tsx

```diff
@@ -0,0 +1,48 @@
+import type { Extension } from '@prezly/slate-commons';
+import { createDeserializeElement } from '@prezly/slate-commons';
+import { CONTACT_NODE_TYPE, isContactNode } from '@prezly/slate-types';
+import React from 'react';
+
+import { composeElementDeserializer } from '#modules/html-deserialization';
+
+import {
+    normalizeContactInfoAttributes,
+    normalizeContactNodeAttributes,
+    parseSerializedElement,
+} from '../press-contacts/lib';
+
+import { InlineContactElement } from './components';
+import type { InlineContactsExtensionParameters } from './types';
+
+export const EXTENSION_ID = 'InlineContactExtension';
+
+export function InlineContactsExtension({
+    renderForm,
```

Comment:
I'm questioning myself, what's the value of extracting the form rendering logic to the external code?

Left the same comment in https://github.com/prezly/prezly/pull/12333 too.

Reply:
One thing I wasn't sure about is where to put the form since it will be used in both the Placeholder extension as well as InlineContacts extension.

Only place I can think of is `src/components` but I am not sure as the form is not a generic component 🤔

Reply:
Hmm, maybe `src/modules/components` could work?